platforms/documentation/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_dependencies.adoc

```diff
@@ -82,6 +82,9 @@ include::sample[dir="snippets/dependencyManagement/attributeMatching/groovy",fil
 ====
 
 Configurations can inherit dependencies from other configurations by extending from them.
+
+_Note:_ Configurations can only extend configurations within the same project.
```

Comment:
We should use `NOTE:`, as this gets transformed specially in the generated HTML ([example](https://docs.gradle.org/current/userguide/dependency_resolution.html#sec:programmatic_api))